source/develop/release-management/features/virt/nvram-data.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Secure boot NVRAM data persistence
+category: feature
+authors: tgolembi,mzamazal
+feature_name: NVRAM data persistence
+feature_modules: engine,vdsm
+feature_status: In development
+---
+
+# Secure boot NVRAM data persistence
+
+## Summary
+
+This feature allows retaining Secure Boot NVRAM data across VM restarts.
+
+## Owner
+
+*   Name: Milan Zamazal
+*   Email: mzamazal@redhat.com
+
+## Description
+
+Secure boot non-volatile memory may contain keys inserted there by the guest OS.  They are used for validation of the software components loaded during boot.  In order to perform such a validation, the recently stored keys must be available also on the next VM start.  This feature adds the required functionality.
+
+All the problems, security considerations, limitations, and implementation options are very similar to [TPM data persistence](tpm-device.md).  The implementation is also almost the same.  This feature page discusses only significant differences to TPM.
+
+Unlike TPM, where libvirt and QEMU use external software to provide TPM emulation, NVRAM data is handled directly by libvirt and QEMU.
+
+## Prerequisites
+
+There are no special prerequisites for this feature.
+
+## User Experience
```

Comment:
this is about the persistence of the NVRAM data but as it may be what users would find when searching for secure boot in oVirt, it worth mentioning that when importing VMs with secure boot from external systems (mostly vsphere), we don't bring this data from those systems

Reply:
Done (and a couple of typos corrected).